src/resources/resourceTypes.js

```diff
@@ -1,14 +1,13 @@
 const RESOURCE_TYPES = {
-  'vital-signs': 'Vital Signs',
   CarePlan: 'Care Plans',
-  Category: 'Categories',
   Condition: 'Conditions',
   DiagnosticReport: 'Diagnostic Report',
   Encounter: 'Encounters',
   Immunization: 'Immunizations',
+  laboratory: 'Lab Results',
   MedicationRequest: 'Medication Requests',
   Procedure: 'Procedures',
-  laboratory: 'Lab Results',
+  'vital-signs': 'Vital Signs',
```

Comment:
This change indicates that we want to use default ordering of Objects, which we don't want to rely on.

...If this is kept _unsorted_, it will be easier to detect that we are applying sorting, and: ensure we implement sorting _explicitly_ and in the right place.

Reply:
Ok, I can revert this.